lib/src/kvs/cache.rs

```diff
@@ -1,3 +1,4 @@
+use crate::idg::u64::U64;
```

Comment:
Or shall we name it `idg::int::U64` so that it reads like a "u64 integer ID generator", where idg stands for "ID generator".